src/crons/delete_old_data_cron.ts

```diff
@@ -0,0 +1,21 @@
+import { Bindings } from "../bindings";
+
+
+async function delete_old_data_cron(env: Bindings) {
+  const response = await env.DB.prepare(`
+      DELETE
+      FROM requests
+      WHERE created_at < datetime('now', ?1)
+  `)
+    .bind(env.DELETE_OLD_DATA_BEFORE)
```

Comment:
what happens if this environment variable is not set for some reason?

Reply:
If not set it should not do anything imo

Reply:
I deliberately left it in a state where it will generate an SQL error, making it easier to troubleshoot by examining the logs.